fastcrypto-zkp/src/unit_tests/verifier_tests.rs

```diff
@@ -212,3 +236,150 @@ fn test_multipairing_with_processed_vk() {
+            133, 232, 141, 167, 190, 26, 129, 90, 57, 4, 246, 59, 153, 125, 79, 61, 69, 237, 62,
+            32, 229, 203, 14, 23, 176, 185, 98, 182, 46, 157, 100, 213, 188, 130, 95, 229, 113,
+            255, 193, 95, 152, 177, 6, 5, 117, 142, 175, 68, 15, 225, 101, 19, 56, 108, 8, 108,
+            158, 11, 11, 234, 28, 48, 248, 248, 191, 22, 103, 220, 196, 117, 20, 169, 173, 196,
+            205, 27, 45, 133, 76, 15, 210, 41, 30, 1, 64, 183, 246, 211, 79, 49, 195, 203, 108,
+            142, 230, 53, 185, 57, 72, 33, 54, 145, 84, 221, 82, 10, 253, 170, 205, 72, 218, 109,
+            238, 219, 25, 15, 39, 245, 157, 151, 64, 195, 96, 123, 191, 203, 44, 15, 138, 89, 11,
+            78, 233, 7, 26, 155, 218, 149, 50, 33, 127, 137, 170, 178, 253, 78, 45, 80, 95, 71,
+            204, 17, 60, 0, 97, 136, 73, 38, 139, 20, 15, 171, 107, 228, 5, 100, 154, 45, 29, 7,
+            73, 131, 24, 50, 135, 184, 238, 122, 115, 196, 219, 178, 171, 78, 123, 163, 186, 183,
+            250, 0, 90, 5, 90, 61, 210, 107, 71, 135, 254, 17, 181, 133, 2, 0, 0, 0, 0, 0, 0, 0,
+            246, 117, 216, 150, 18, 57, 84, 24, 157, 52, 104, 30, 245, 206, 71, 181, 227, 38, 2,
+            71, 228, 234, 104, 23, 241, 156, 65, 11, 159, 111, 227, 222, 176, 134, 225, 101, 5,
+            108, 2, 33, 106, 14, 18, 17, 74, 155, 65, 13, 118, 128, 94, 144, 97, 147, 200, 205, 68,
+            176, 47, 205, 157, 155, 52, 253, 182, 178, 117, 239, 92, 19, 231, 5, 111, 182, 26, 161,
+            135, 4, 9, 184, 2, 8, 16, 249, 178, 154, 171, 107, 51, 159, 163, 248, 83, 192, 225, 3
+        ],
+        vec![
+            247, 109, 145, 212, 188, 154, 57, 115, 193, 94, 58, 235, 87, 79, 15, 100, 84, 123, 131,
+            143, 149, 10, 243, 93, 185, 127, 7, 5, 196, 33, 75
+        ],
+        vec![
+            207, 67, 33, 174, 120, 198, 30, 222, 247, 157, 208, 196, 178, 230, 196, 164, 140, 36,
+            145, 78, 155, 43, 142, 106, 169, 255, 12, 94, 20, 27, 234, 232, 75, 128, 180, 149, 16,
+            190, 185, 2, 24, 167, 108, 237, 179, 157, 204, 151, 252, 48, 158, 214, 217, 17, 200,
+            173, 101, 151, 94, 8, 27, 81, 192, 137, 201, 90, 112, 234, 109, 213, 22, 202, 9, 201,
+            165, 156, 78, 228, 246, 36, 214, 69, 236, 188, 159, 172, 2, 1, 148, 204, 9, 98, 171,
+            79, 4, 15, 77, 118, 91, 14, 105, 1, 74, 71, 188, 159, 27, 6, 224, 186, 129, 139, 255,
+            242, 165, 31, 66, 78, 62, 186, 50, 91, 81, 78, 13, 168, 140, 78, 10, 174, 57, 146, 49,
+            191, 216, 218, 162, 149, 54, 207, 45, 220, 160, 152, 111, 136, 20, 123, 116, 157, 27,
+            229, 148, 55, 97, 10, 175, 125, 12, 52, 178, 0, 245, 142, 45, 42, 147, 244, 236, 209,
+            66, 8, 163, 20, 88, 56, 4, 221, 42, 59, 194, 131, 236, 0, 222, 1, 236, 247, 137, 56,
+            69, 7
+        ]
+    )
+    .is_err());
+
+    // length of proof is incorrect
+    assert!(verify_groth16_in_bytes(
```

Comment:
bytes used in this test is derived from the previous test https://github.com/MystenLabs/fastcrypto/pull/223/files#diff-d842dec425f6894bd38212716a7896634076a8693bd4621554d48476775b9e91L173

Reply:
It would be great to be able to take the example in the following test, defined as a constant, clone it, edit the bytes, and then call an assert - it would make your comment clear from the code.